### ISSUE TITLE
Question about batch_size and the InfoNCE contrastive loss

### ISSUE THREAD

**Issue by PoissonChasseur** [VMarsocci/X-STARS#2]
I read your paper and found it very interesting. For the InfoNCE contrastive loss, when I looked at the {SimCLR, MoCo-V2, MoCo-V3} papers in particular, it seems that there can be quite significant performance differences depending on the value of the batch_size (since the "negative samples" are based on those of the current batch) (which can be around 1-3%). In {SimCLR, MoCo-V3} in particular, they used a quite large batch_size (like 4096 in MoCo-V3).

In your case, it seems that you used a batch_size = 32/GPU (on 8 GPUs, global batch_size = 256) (or for the ablation study, batch_size = 64/GPU (on 2 GPUs, global = 128)). I was wondering if by any chance you did (at some point) some experiments on the value of the batch_size. I was wondering if, due to the fact that X-STARS uses 2 loss functions (DINO-V1 knowledge distillation loss and MSAD loss), the value of the batch_size (during the pre-training) might have less impact on the downstream performances (e.g., small performance variation between small batch_sizes like 128 vs larger batch_sizes like 256 or highter)?

Additionally, in your paper, for the MSAD loss, you identified (based on cross-validation) the values ​​of "label smoothing α = 0.3 and a weight λ = 0.1". I was wondering whether, during these experiments, the choice of these values ​​had or not significant impacts on the obtained performances.

**Comment by VMarsocci** [comment]
Hi thanks a lot for your question. 

We didn't observe big changes when changing the batch size (we agree with you on why).

Similar answer for the values of \alpha and \lambda.
